app/controllers/admin/ReportsController.php

```diff
@@ -45,13 +45,13 @@ public function exportAssetReport()
             Lang::get('general.model_no'),
             Lang::get('general.name'),
             Lang::get('admin/hardware/table.serial'),
+            Lang::get('general.status'),
             Lang::get('admin/hardware/table.purchase_date'),
             Lang::get('admin/hardware/table.purchase_cost'),
             Lang::get('admin/hardware/form.order'),
             Lang::get('admin/hardware/form.supplier'),
             Lang::get('admin/hardware/table.checkoutto'),
-            Lang::get('admin/hardware/table.location'),
-            Lang::get('general.status')
```

Comment:
Status and location are kinda different though. Location is physical location. Status is the status, unless it's deployed to someone, in which case status is the person's name that it's been checked out to.